recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - tomli
   run:
     - python >=3.9
-    - dask ==2024.2.1
+    - dask-core ==2024.3.0
```

Comment:
Switching to `dask-core` instead of the `dask` metapackage to avoid circular dependencies